source/patterns/@aws-solutions-constructs/aws-sqs-pipes-stepfunctions/README.md

```diff
@@ -0,0 +1,106 @@
+# aws-sqs-pipes-stepfunctions module
+<!--BEGIN STABILITY BANNER-->
+
+---
+
+![Stability: Experimental](https://img.shields.io/badge/stability-Experimental-important.svg?style=for-the-badge)
+
+---
+<!--END STABILITY BANNER-->
+
+| **Reference Documentation**:| <span style="font-weight: normal">https://docs.aws.amazon.com/solutions/latest/constructs/</span>|
+|:-------------|:-------------|
+<div style="height:8px"></div>
+
+| **Language**     | **Package**        |
+|:-------------|-----------------|
+|![Python Logo](https://docs.aws.amazon.com/cdk/api/latest/img/python32.png) Python|`aws_solutions_constructs.aws_sqs_pipes_stepfunctions`|
+|![Typescript Logo](https://docs.aws.amazon.com/cdk/api/latest/img/typescript32.png) Typescript|`@aws-solutions-constructs/aws-sqs-pipes-stepfunctions`|
+|![Java Logo](https://docs.aws.amazon.com/cdk/api/latest/img/java32.png) Java|`software.amazon.awsconstructs.services.sqspipesstepfunctions`|
+
+## Overview
+This AWS Solutions Construct implements an AWS SQS queue whose messages are passed to an AWS Step Functions state machine by and Eventbridge pipe.
```

Comment:
This AWS Solutions Construct implements an Amazon SQS queue whose messages are passed to an AWS Step Functions state machine by an Amazon EventBridge pipe.